### PR TITLE
XPK blueprint updates to make it more compatible with the tool

### PR DESCRIPTION
**Pull request by ankitkinra**
# Minor fixes for xpk compatibility 

* Update the naming convention of the node pools
* Name the config map with deployment name

### CODE REVIEW

File: community/examples/xpk-gke-a3-megagpu.yaml [hunk 1]
```diff
@@ -70,25 +72,40 @@ deployment_groups:
...
+      name: system-pool
+      machine_type: "e2-standard-16"
+      autoscaling_total_min_nodes: 2
+      autoscaling_total_max_nodes: 8
+      initial_node_count: 2
+      zones: [$(vars.zone)]
+      host_maintenance_interval: PERIODIC
+    outputs: [instructions]
+
+  - id: a3_megagpu_pool_np_0
```
Comment (pawloch00) [inline community/examples/xpk-gke-a3-megagpu.yaml:88]: why do we have two nodepools?

**Comment by pawloch00** [comment]
Should TAS be left here? @ankitkinra

File: community/examples/xpk-gke-a3-megagpu.yaml (hunk 1 above)
Comment (ankitkinra) [reply community/examples/xpk-gke-a3-megagpu.yaml:88]: The second modepool has no default taints and can handle system software which has not been modified by our team.

File: community/examples/xpk-gke-a3-megagpu.yaml [hunk 2]
```diff
@@ -33,6 +33,7 @@ deployment_groups:
     source: modules/network/vpc
     settings:
       subnetwork_name: xpk-gke-a3-megagpu-subnet
+      mtu: 8244
```
Comment (annuay-google) [inline community/examples/xpk-gke-a3-megagpu.yaml:36]: Curious, any reason to set the MTU to 8244?

**Review by annuay-google — APPROVED** [review]
LGTM, comment is non blocking

File: community/examples/xpk-gke-a3-megagpu.yaml (hunk 2 above)
Comment (ankitkinra) [reply community/examples/xpk-gke-a3-megagpu.yaml:36]: this is from the best practices from GKE from https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#create-vpcs-subnets

**Comment by ankitkinra** [comment]
> Should TAS be left here? @ankitkinra

We can potentially update in a follow up PR

**Pull request merged**